doc/org.eclipse.python4capella.doc/doc/user/general/Run.textile

```diff
@@ -0,0 +1,43 @@
+h1. Run a Pyhton script
+
+This section explains how you can run your Python4Capella script either within Capella or from the command line.
+
+h2. Within Capella
+
+You will need to enable Development capabilities if it's not already done:
```

Comment:
You should create a section "Enable Development Capabilities" en just refers to this section here.